rholang/src/main/scala/coop/rchain/rholang/interpreter/normalize.scala

```diff
@@ -41,7 +41,7 @@ object GroundNormalizeMatcher {
   def normalizeMatch(g: AbsynGround): Expr =
     g match {
       case gb: GroundBool   => BoolNormalizeMatcher.normalizeMatch(gb.bool_)
-      case gi: GroundInt    => GInt(gi.integer_)
+      case gi: GroundInt    => GInt(gi.long_.toLong)
```

Comment:
My only concern is this might throw, but as I understand we catch that and report as `UnrecognizedInterpreterError` ?

Reply:
ummm... right... forgot about that :joy:

Yes, it will be caught and reported just as you've said.
I'll add a TODO for handling the error purely.